uaa/src/test/java/org/cloudfoundry/identity/uaa/integration/OrchestratorZoneControllerIntegrationTests.java

```diff
@@ -308,27 +312,28 @@ private void testAdminClientSecretAsSpaceAndEmpty(OrchestratorZoneRequest orches
             String.class);
         assertEquals(getResponse.getStatusCode(), HttpStatus.BAD_REQUEST);
         assertEquals(APPLICATION_JSON_UTF8, getResponse.getHeaders().getContentType());
-        String expected  =  JsonUtils.writeValueAsString(
-            new OrchestratorErrorResponse("The adminClientSecret field cannot contain" +
-                                          " spaces or cannot be blank."));
-        assertEquals(expected, getResponse.getBody());
+        assertTrue(getResponse.getBody().contains("orchestratorZoneRequest.parameters.adminClientSecret " +
```

Comment:
this one 'orchestratorZoneRequest', i think we decided to remove this as this name is not part of the actual request payload.

Reply:
Done. Thanks!